modules/conflict_resolver/test/TestPlan.md

```diff
@@ -5,6 +5,7 @@
     2. Menu item loads the module page
     3. Check for existence of 'Resolving conflicts' permission checkbox on 'User Accounts > Edit User'
        Verify that the permission works as expected (hides menu item and blocks page access)
+    4. Check that access will be blocked for users who dont have either the 'Data Dictionary (Legacy): Edit Parameter Type Descriptions' or 'Data Dictionary (Legacy): View Parameter Type Descriptions'
```

Comment:
Can we change the owner of these permissions codes in the `permissions` table from the `data_dict` module to the `dictionary` module? Both modules use the same 2 permissions.

Reply:
just to confirm: these permissions will now read as 'Dictionary: Edit Parameter Type Descriptions' and  'Dictionary: View Parameter Type Descriptions